### PR TITLE
Chore/images list e2e

### PR DESCRIPTION
**Pull request by santiagonoguez**

### CODE REVIEW

File: test/e2e/template-editor/pages/components/imageComponentPage.js [hunk 1]
```diff
@@ -26,6 +28,27 @@ var ImageComponentPage = function() {
...
+  this.getBrokenLinks = function() {
+    return element.all(by.xpath('//div[@class="image-thumbnail"]/div[2]'));
+  }
+
+  this.removeImageRow = function (selectorLabel) {
+    var removeLink = element(by.xpath(
+      '//div[@class="image-remove"][..//div[contains(text(), "' + selectorLabel + '")]]/a'
+    ));
+
+    helper.wait(removeLink, 'Image Row Remove');
```
Comment (fjvallarino) [inline test/e2e/template-editor/pages/components/imageComponentPage.js]: @santiagonoguez in general the `*Page.js` files only contain logic for declaring selectors and related. All `wait` and `clickWhenClickable` calls should go into `image.js`

Comment (fjvallarino) [reply test/e2e/template-editor/pages/components/imageComponentPage.js]: @santiagonoguez having said that, it's true that in some cases this pattern is not followed (mainly in some common `*Page.js` files). Up to you if you prefer to keep it this way.

Comment (santiagonoguez) [reply test/e2e/template-editor/pages/components/imageComponentPage.js]: @fjvallarino I'll change it, I think it's cleaner that way, thanks !

Comment (santiagonoguez) [reply test/e2e/template-editor/pages/components/imageComponentPage.js]: @fjvallarino changes applied !

**Pull request merged**